elfpy/markets/hyperdrive/yieldspace_pricing_model_sol.py

```diff
@@ -0,0 +1,111 @@
+"""The Hyperdrive pricing model"""
+from __future__ import annotations
+
+from fixedpointmath import FixedPoint
+
+# Let the variable names be the same as their solidity counterpart so that it is easier to compare
+# the two.  We can make python wrappers that just call these methods that have better variable names
+# that conform to python standards.
+# pylint: disable=invalid-name
+# pylint: disable=too-many-arguments
+
+ONE_18 = FixedPoint("1")
+
+
+def modified_yield_space_constant(
```

Comment:
This exists: https://github.com/delvtech/elf-simulations/blob/0341920289309f93f9ade98cb89db2c1477d3d7d/elfpy/markets/hyperdrive/yieldspace_pricing_model.py#L698